docs/source/installation/win.rst

```diff
@@ -1,13 +1,73 @@
 Windows
 =======
 
-First, you need to install Chocolatey, which is a package manager for Windows
-systems.  Please refer to `this link <https://chocolatey.org/install>`_ for
+There are two simple ways to download manim's depedencies, using the popular package
```

Comment:
Non-blocking: This document has become a bit long, how about using https://pypi.org/project/sphinx-tabs/, one for chocolatey and the other for scoop?

Reply:
Sure, should it be done in this PR or a new one? Since it would require a dependency to be added.

Reply:
I think it should be fine with this PR itself.